### DISCUSSION TITLE
Is todolist example representative of Mix?

### DISCUSSION THREAD

**Discussion by romanr** [btwld/mix#494] [General]
None of interactive elements have pressed state, which is a serious issue.  
Looking at the code, to Save there has to be created a "TodoButton" class, even if it's only used once. It's creating more work rather than making it easier?

**Comment by tilucasoli** [comment]
Hey @romanr. The `examples` folder is a bit dated and has an old instance of Mix usage. However, I could help you understand what's new and the best practices. We are currently working on improving our examples, by the way.

About the `TodoButton`, the entire `todo_list` project follows best practices for larger projects, and componentization is just one of them.
